mlflow/pyfunc/__init__.py

```diff
@@ -709,7 +729,10 @@ def _warn_dependency_requirement_mismatches(model_path):
 
 
 def load_model(
-    model_uri: str, suppress_warnings: bool = False, dst_path: str = None
+    model_uri: str,
+    suppress_warnings: bool = False,
+    dst_path: str = None,
+    support_decimal: bool = False,
```

Comment:
Is there a way to accomplish the goal of providing conditional decimal enforcement without adding this flag to the `mlflow.pyfunc.load_model()` API? I'd rather not extend this API for a niche use case.

Reply:
I personally don't think we need this flag at all. Can we just always enable this conversion? I don't see any real use case of setting that to false (or only add that flag later when we confirm it's a real use case).

@dbczumar @tomasatdatabricks what do you think?

Reply:
I think tom provided 2 approach to me 
<img width="819" alt="image" src="https://user-images.githubusercontent.com/109561646/187545783-e5c14724-6be8-4a49-bdcf-371f19918bb4.png">

The other option I could think on top of my head is use environment variable

https://github.com/mlflow/mlflow/blob/master/mlflow/environment_variables.py

I am not sure which approach is more preferrable, let's have a quick agreement.

Reply:
I agree with @yxiong here

Reply:
Reverted back the change, now we always enable the cast.